source/opt/mem_pass.cpp

```diff
@@ -97,6 +97,8 @@ Instruction* MemPass::GetPtr(uint32_t ptrId, uint32_t* varId) {
   Instruction* ptrInst = get_def_use_mgr()->GetDef(*varId);
   Instruction* varInst;
 
+  if (ptrInst->opcode() == SpvOpConstantNull) return ptrInst;
```

Comment:
Also set `*varId` to `0`, please.  So the caller in the SSA re-writer doesn't inadvertently consider it a target var.

Reply:
done